### DISCUSSION TITLE
How to view and edit ".bat" files?

### DISCUSSION THREAD

**Discussion by aisbergde** [gsantner/markor#1843] [Q&A] [answered]
I have some files with the extension ".bat" in my notes foldet, but I can't view or edit them with Markor. If I press on them like I press on normal files, then a dialog opens, asking me "copy to" and giving me some targets to choose between. But I just want to open. I have no idea where this special behavior is defined.

But I can use other editors to see and edit the content of bat files.

Where this behavior is defined? It looks like some special handling in Markor.

**Comment by gsantner** [comment, marked as answer]
There is a settings option "open with markor" and there you can add .bat so when you click on it in the markor file manager, Markor opens it diretcly in editor.
